components/raftstore/src/store/worker/pd.rs

```diff
@@ -191,11 +191,13 @@ impl Default for StoreStat {
 pub struct PeerStat {
     pub read_bytes: u64,
     pub read_keys: u64,
-    pub last_read_bytes: u64,
-    pub last_read_keys: u64,
-    pub last_written_bytes: u64,
-    pub last_written_keys: u64,
-    pub last_report_ts: UnixSecs,
+    pub last_region_report_read_bytes: u64,
+    pub last_region_report_read_keys: u64,
+    pub last_region_report_written_bytes: u64,
+    pub last_region_report_written_keys: u64,
+    pub last_region_report_ts: UnixSecs,
+    pub last_store_report_read_bytes: u64,
```

Comment:
IMO, difference between `last_region_report_read_bytes` and `last_store_report_read_bytes` is that their interval are different. Is it true? Would you like to add some comments?

Reply:
updated.